Tests/BowTests/Typeclasses/MonadLaws.swift

```diff
@@ -17,10 +17,10 @@ class MonadLaws<F> {
     }
     
     private static func leftIdentity<Mon, EqF>(_ monad : Mon, _ eq : EqF) where Mon : Monad, Mon.F == F, EqF : Eq, EqF.A == Kind<F, Int> {
-        property("Monad left identity") <- forAll { (a : Int, b : Int) in
-            let f = { (_ : Int) in monad.pure(a) }
-            return eq.eqv(monad.flatMap(monad.pure(b), f),
-                          f(b))
+        property("Monad left identity") <- forAll { (a : Int, f : ArrowOf<Int, Int>) in
+            let g = f.getArrow >>> monad.pure
```

Comment:
What's that operator?

Reply:
`andThen`, reverse of `<<<` or `compose`. Some other libraries already use it.